crates/ruma-serde-macros/src/lib.rs

```diff
@@ -27,6 +27,7 @@ mod util;
 /// of the struct, this simple implementation will be generated:
 ///
 /// ```ignore
+/// # // TODO: "ignore" this doctest until it is more extensively documented. (See #541)
```

Comment:
I don't really understand this TODO comment.. This `ignore` is probably fine.

Reply:
I'm providing reasons as to why each doctest is ignored, it's arguably bad to leave it ignored as-is, but at least i'm providing reasons as to why its ignored, so someone in the future can pick up on it.

Reply:
I get that, but I don't really understand how "more extensively" documenting something here would resolve the `ignore`.

Reply:
There's some blurbs of doctest underneath this comment, I know that writing a more "complete" doctest will resolve the run and compilation errors, but at the moment i dont know how to do it, ergo "more extensively documented"